into_repo.sh

```diff
@@ -0,0 +1,3 @@
+# override xenctrl.system to obtain an opam repository without internal xenserver dependencies
+curl -L https://raw.githubusercontent.com/xapi-project/ocaml-xen-lowlevel-libs/master/opam > packages/xs/xenctrl.system/opam
+echo 'archive: "https://github.com/xapi-project/ocaml-xen-lowlevel-libs/archive/master.tar.gz"' > packages/xs/xenctrl.system/url
```

Comment:
Another option would be to remove `xs/xenctrl.system`, and put these into `xs-extra/xenctrl.master` to reflect the different package version. However, this solution is simpler and maybe safer as it does not delete files, just overwrites them. And soon we will move out `xenctrl.system` from the default `xs-opam` repo anyway (see my comment below).